server-src/src/main/java/com/itt/kmt/model/ArticleApproval.java

```diff
@@ -0,0 +1,56 @@
+package com.itt.kmt.model;
+
+import lombok.Data;
+
+/**
+ * This class represents a ArticleApproval.
+ * 
+ * @author Ashish Y
+ */
+@Data
+public class ArticleApproval {
+
+    /**
+     * approval Status of the Article.
+     */
+    private Boolean approvalStatus =  false;
+    /**
+     * approvers, semicolon separated userIds.
+     * who can approve an article
+     */
+    private String approvers;

```

Comment:
This should be a single approver and should be a reference with User who has been chosen to approve the Knowledge Article.